scripts/grunt.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 echo "Installing Grunt & Grunt CLI by NPM"
-npm install -y -g grunt-cli
```

Comment:
Yep, it didn't seem to do anything... Probably shouldn't have been in this PR, but oh well!